### PR TITLE
fix!: remove liquidation targets query

### PR DESCRIPTION
**Pull request by toteki**
## Description

Removes a query which was capable of seriously slowing down nodes on testnet (i.e. freeze 40sec per usage)

closes: #827 

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added appropriate labels to the PR
- [x] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### API Breaking
+TODO
```
Comment (toteki) [inline CHANGELOG.md:50]:
```suggestion

- [1028](https://github.com/umee-network/umee/pull/1028) Remove liquidation targets query
```

**Comment by toteki** [comment]
Might need someone to run `make-proto-gen` on their machine for this one. My computer is failing to produce the required changes (removing liquidation targets query from `.pb.gw.go`) when I do it locally.

**Comment by toteki** [comment]
> Looks like the `.pb.gw.go` file has been updated - is there more to be removed?
> 
> Also, is this query still necessary for liquidation targeting? @toteki

I did the file manually to compile so there might be diffs if gen were run properly.

We will need some other way to do this query on the liquidators side, but without exposing it on real nodes because of the huge burden it creates. Maybe reintroduce later with a flag or enable?

**Comment by toteki** [comment]
Maybe we can enable the query only when a runtime flag is set, so nobody has it by accident.

In that's the case, then this PR with the proto removal can be closed, and a new one with the flag opened.

**Comment by adamewozniak** [comment]
@toteki can we paginate this with an upper limit instead?

That way all clients could expect the same type of queries to work for all nodes, and some of the burden is reduced

**Pull request closed without merging**